docs/sources/gard.md

```diff
@@ -1,17 +1,90 @@
+and tried to determine the best predicates to use for mappings. The results are in the [`gard.sssom+` sheet](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=1504531712). 
+This only applies to those rows where `gard_mondo_mapping_type`==`proxy_new`.
+1. **GARD::Orphanet**: `skos:exactMatch`: Where `DataSource`==`Orphanet`, value from the `SourceID` column is an Orphanet ID.
+2. **GARD::Orphanet**: `skos:broadMatch`: There is a `ParentOrphaCode` column. Any IDs from that column are considered broad matches.
+3. **GARD::OMIM**: `skos:exactMatch`: I’m unsure why they called it "`Orphanet+OMIM`", but based on what Yanji said, where 
+`DataSource`==`Orphanet+OMIM`, the value in the `SourceID` column is an OMIM ID.
+4. **GARD::OMIM**: `skos:narrowMatch`: These come from the `OmimMember` column. Rows has 0 to many MIM#s in that column. 
+Exception 1: when `DataSource`==`Orphanet+OMIM`. In that case, there is only 1 item in `OmimMember`, and it is equal to what 
+is in the `SourceID` column and considered an exact match. Exception 2: when `DataSource`==`GARD`, these are considered 
+related matches. Yanji told Joe that these rows are in development, so perhaps they might be exact, narrow, or perhaps 
+even broad.
+5. **GARD::OMIM**: `skos:relatedMatch`: For rows where `DataSource`==`GARD` (n=10), some of these rows have values in the 
+`OmimMember` column (n=5). In all these cases, it’s just 1 MIM#. Yanji mentioned in email that these rows were 
+tentative / being worked on. So I felt that without asking or taking a closer look, I was unsure if these would be 
+narrow or exact matches.
 
+#### Misc notes
+1. Where `DataSource` `Orphanet+OMIM`: what IDs are these? OMIM or Orphanet? **Answer: OMIM**
+2. Do any rows where `DataSource`==`Orphanet+OMIM` have multiple OMIM mappings in the `OmimMember` column? **Answer: No**
+3. What values are in the `DataSource` column? **Answer: `GARD`, `Orphanet`, and `Orphanet+OMIM`**
 
+#### Discussion
+**Joe 2023/04/23**  
+New results:  
+**Unmapped GARD terms**: 181 of 12004 (1.51%)  
+**Mapped GARD terms**: 11823 of 12004 (98.49%)  
 
+I fixed the issue where I was accidentally mapping to ORDO IDs that should have been OMIM IDs. I also found some new 
+columns to map to. I versioned the [spreadsheet](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=1607652739), 
+then updated it. The main sheet we want to look at now [is `gard.sssom+`](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=1504531712).
+
+I think you guys should review my notes in the readme to see how I determined mappings. I also added comments to some 
+column headers explaining its contents and how I interpreted those fields.
+
+I imagine you guys will be happy by this much higher mapping %! But also you will wonder why the number of unmapped 
```

Comment:
We are happy - if these mappings are all "exact"! document above how you determined that the mappings you  added are exact.